### PR TITLE
Prevent NullPointerException when joining a world

### PR DESCRIPTION
**Pull request by Chanakan55991**
This simple pr fixed a NullPointerException that occurred to me when I tried joining a world with nofall enabled.

### CODE REVIEW

**Comment by Chanakan55991** [comment]
also one problem related to nofall module is you will get lagback when you go under anything that is 2 block high on packetmodify mode, since it seems to add player y position by 1

File: src/main/java/com/paragon/client/systems/module/impl/movement/NoFall.java [hunk 1]
```diff
@@ -91,6 +91,9 @@ public void onTick() {
 
     @Listener
     public void onPacketSent(PacketEvent.PreSend event) {
+        if(mc.player == null || mc.playerController == null) {
```
Comment (SooStrator1136) [inline src/main/java/com/paragon/client/systems/module/impl/movement/NoFall.java:94]: You should use nullCheck()

Comment (Chanakan55991) [reply src/main/java/com/paragon/client/systems/module/impl/movement/NoFall.java:94]: I forgot so much about minecraft forge development lol

Comment (SooStrator1136) [reply src/main/java/com/paragon/client/systems/module/impl/movement/NoFall.java:94]: Just change it, commit and ill merge it

**Pull request merged**